trikot-viewmodels-declarative-flow/viewmodels-declarative-flow/src/commonMain/kotlin/com/mirego/trikot/viewmodels/declarative/components/factory/VMDComponents.kt

```diff
@@ -38,6 +45,15 @@ object VMDComponents {
                 VMDTextViewModelImpl(coroutineScope)
                     .apply { text = content }
                     .apply(closure)
+
+            fun withContent(
+                contentFlow: Flow<String>,
```

Comment:
As discussed, `Content` types should not ben enclosed in `Flow`, the parent view model should be responsible of handling the Flow binding and content should just be the raw type. IE, instead of creating `Text.withContent(Flow<String>)`, one should create `Text.withText(String)` followed by `textViewModel.bindtext(Flow<String>)`.